src/model-helpers/Profile.js

```diff
@@ -0,0 +1,128 @@
+import {
+  buildThing,
+  createThing,
+  getStringNoLocale,
+  getThing,
+  getWebIdDataset,
+  removeStringNoLocale,
+  removeThing,
+  saveSolidDatasetAt,
+  setThing
+} from '@inrupt/solid-client';
+import { RDF_PREDICATES } from '../constants';
```

Comment:
Since these functions are all related to the currently signed in user, can we put them in the `SignedInUserContext`? You can look at `UserList` and `DocumentList` for models of how to do that.

Reply:
I have placed the functions being used in `Profiles` as part of `SignedInUserContext`. I think that's what this comment refers to.